www/source/docs/index.html.slim

```diff
@@ -14,12 +14,12 @@ p If you're just getting started and want a quick introduction, then we recommen
 .row
   .columns.medium-6
     .box-white-brdr.strict-center.
-      p Complete a short interactive demo
-      a.btn.try-demo href="#" Try the Demo
+      p.txt-margin Complete a short interactive demo
+      a.btn.try-demo href="http://inspec.io/tutorial" Try the Demo
```

Comment:
Can we make this a relative URL (i.e. `href="/tutorial"`) so it works in local/acceptance environments?